lib/ansible/executor/powershell/psrp_fetch_file.ps1

```diff
@@ -0,0 +1,41 @@
+# (c) 2025 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+using namespace System.IO
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $true)]
+    [string]
+    $Path,
+
+    [Parameter()]
+    [int]
+    $BufferSize
```

Comment:
Default?

Reply:
We always pass it in but I suppose I can set a sane default here so it doesn't become `0` if we ever forget about it.

Reply:
I just marked it as Mandatory as the value is derived at runtime based on the server configured envelope size.